lib/stdlib/src/lists.erl

```diff
@@ -97,7 +97,18 @@ An example of a typical ordering function is less than or equal to: `=</2`.
 %% Shadowed by erl_bif_types: lists:keyfind/3
 -doc """
 Searches the list of tuples `TupleList` for a tuple whose `N`th element compares
-equal to `Key`. Returns `Tuple` if such a tuple is found, otherwise `false`.
+equal to `Key`.
+
+Returns `Tuple` if such a tuple is found; otherwise, returns `false`.
+
+_Examples:_
+
+```
```

Comment:
```suggestion
```erlang
```

Reply:
Consistency is good, but I don't think there is any technical reason to have `erlang`. Both ExDoc and shell_docs_test assume that if no language is given, it is `erlang`.